### PR TITLE
Feature/low mem clj

### PR DESCRIPTION
**Pull request by muep**

### CODE REVIEW

File: _posts/2022-03-07-running-clojure-on-constrained-memory.md
Comment (tatut) [inline _posts/2022-03-07-running-clojure-on-constrained-memory.md:179]: an OOME wouldn't terminate the whole JVM program, just the thread, but might leave your program in an inconsistent state (especially if you are using mutable objects).

Comment (muep) [reply _posts/2022-03-07-running-clojure-on-constrained-memory.md:179]: Thank you for pointing this out. I wasn't fully conscious that it would only end the thread. But there is a big difference between a terminated program and broken but running program, so should fix this.

**Review by mattiuusitalo — APPROVED** [review]
This post is superb! It has been waiting for approval / publication far too long.

**Pull request merged**
